lms/exceptions.py

```diff
@@ -13,7 +13,19 @@ class LTILaunchError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
     pass
 
 
+class LTIContentItemError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
```

Comment:
The "Content selection" mode is a "kind" of LTI launch, so I think this Exception class may be unnecessary...